certified-connectors/Tribal/apiDefinition.swagger.json

```diff
@@ -3303,6 +3295,82 @@
           }
         }
       }
+    },
+    "/": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@mszy-tribal empty schema is not allowed. please define a proper schema.

Reply:
I cannot include a schema for the 2XX responses as this action is a custom http request and we don't know what they user is going to look for.

We plan in inform our customers to use the "Parse JSON" action alongside consulting our API documentation.